src/iris/utils.py

```diff
@@ -9,17 +9,29 @@
 from gevent import sleep
 import datetime
 import ujson
+import os
 from . import db
+from iris.config import load_config
 import re
 import msgpack
 import logging
+from random import randrange
 
 logger = logging.getLogger(__name__)
 
+config = {}
+if 'IRISPYTEST' not in os.environ:
+    config = load_config()
```

Comment:
why is this necessary?

Reply:
The tests are not run with a config file and this is the only thing in tests that calls load_config() without this load_config() hits a filenotfound error and bails on execution.

Reply:
I'm just confused how this ever worked then. Did something change in this PR that required this?

Reply:
load_config() wasn't being called by the unit tests before until the bucket assignment in utils (and by extension incident creation) used it starting with this PR.